appveyor.yml

```diff
@@ -11,12 +11,11 @@ environment:
     CABOPTS:  "--store-dir=C:\\SR --http-transport=plain-http"
     DOCTEST: YES
   matrix:
-    - GHCVER: 7.8.4.1
-    - GHCVER: 7.10.3.2
     - GHCVER: 8.0.2
     - GHCVER: 8.2.2
     - GHCVER: 8.4.4
     - GHCVER: 8.6.3
```

Comment:
Perhaps also GHC 8.6.5, rather than 8.6.3?

Reply:
GHC 8.10.1 is also available, I'm just waiting on some details to upload cabal to support it

Reply:
Done.